src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -502,18 +502,17 @@ export class InputTimePicker
 
   private async loadDateTimeLocaleData(): Promise<void> {
     const { effectiveLocale } = this;
+    let dayjsLocale = effectiveLocale.toLowerCase();
```

Comment:
Should we call this normalizedLocale to be consistent with the other code? Maybe a function to normalize the locale would promote consistency since `toLowerCase` is used twice for the same thing.

This logic is repeated, I think a function would be beneficial. Something that converts the effectiveLocale to the format we expect.

```ts
const normalizedLocale = effectiveLocale.toLowerCase();

    if (normalizedLocale === "en-us") {
      effectiveLocale = "en";
    }
    if (effectiveLocale === "pt-PT") {
    if (normalizedLocale === "pt-pt") {
      effectiveLocale = "pt";
    }
    if (effectiveLocale === "no") {
    if (normalizedLocale === "no") {
      effectiveLocale = "nb";
    }
```

Reply:
Good call, I'll follow this up with just such a function.  I agree it would be nice to handle this logic all in one place.

Reply:
I think we could add another context to [`locale.getSupportedLocale`](https://github.com/Esri/calcite-components/blob/master/src/utils/locale.ts#L153) for this since it already tackles normalization. Let's sync up to do this separately.

On related note, I was thinking on adding a module to abstract away `dayjs` to not have to do this per component.